modules/test/playwright/pages/document-library-web/documentLibrary.page.ts

```diff
@@ -37,9 +49,33 @@ export class DocumentLibraryPage {
 		await this.page.getByRole('menuitem', {name: 'Edit'}).click();
 	}
 
+	async new() {
```

Comment:
I prefer the pattern `createSomething` or goToCreate if navigating to the page of creation.

![image](https://github.com/liferay-content-management/liferay-portal/assets/67901/3843cb77-c62d-450e-bf7e-98165f0dce52)

Reply:
I see @boton, but here I think it cannot be applied. I want to click the New button in the management toolbar, that shows the dropdown where the option for Create AI Image could be present or not, that is what I want to test.

Reply:
Changed! I replaced it with `openNewButton().`